indexing/src/main/resources/org/allenai/common/indexing/indexing.conf

```diff
@@ -1,136 +1,88 @@
-org.allenai.common.indexing: {
+include "mappingHelpers.conf"
 
-  stoplist: {
+org.allenai.common.indexing {
+  stoplist {
```

Comment:
Since there's only one entry here, you could name the top-level with suffix `.stoplist` and remove one level of indentation.